.travis.yml

```diff
@@ -10,3 +11,30 @@ python:
   - "nightly" # currently points to 3.6-dev
```

Comment:
I think it's okay to replace `3.5-dev` with `3.6` in this PR, so that we'd have the whole version coverage.

Reply:
P.S. nightly points to 3.8-dev now